setup_f5_eks.sh

```diff
@@ -348,6 +364,74 @@ fi
 echo -e "\nInstalling Fusion 5.0 Helm chart ${CHART_VERSION} into namespace ${NAMESPACE} with release tag: ${RELEASE}"
 
 
+INGRESS_VALUES=""
+if [ "${INGRESS_HOSTNAME}" != "" ] && [ "${DEPLOY_ALB}" == "1" ]; then
+
+  #Creating required ALB policy and attaching it to the Cluster Node Group
+
+  wget https://raw.githubusercontent.com/kubernetes-sigs/aws-alb-ingress-controller/v1.1.5/docs/examples/iam-policy.json -O alb-policy.json
+
+  NODE_GROUP_ARN=$(aws --profile "${AWS_ACCOUNT}" --region "${REGION}" cloudformation describe-stack-resource --stack-name eksctl-${CLUSTER_NAME}-nodegroup-standard-workers --logical-resource-id NodeInstanceRole --query 'StackResourceDetail.PhysicalResourceId' --output text)
+
+  POLICY_ARN=$(aws --profile "${AWS_ACCOUNT}" --region "${REGION}" iam create-policy --policy-name eksctl-${CLUSTER_NAME}-alb-policy --policy-document file://alb-policy.json --query 'Policy.Arn' --output text)
+
+  aws --profile "${AWS_ACCOUNT}" --region "${REGION}" iam attach-role-policy --role-name ${NODE_GROUP_ARN} --policy-arn ${POLICY_ARN}
+
+
+  #Installing ALB
+
+  echo -e "\nInstalling Alb Ingress Controller"
+
+  helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.com
+  helm install incubator/aws-alb-ingress-controller --set autoDiscoverAwsRegion=true --set autoDiscoverAwsVpcID=true --set clusterName=${CLUSTER_NAME} --generate-name
```

Comment:
This is installing into the default namespace, how are we dealing with the case that the script may be run to target multiple namespaces, we can't have more than one alb controller running in the cluster, or all the controllers will try and satisfy the ingress resourecs.

Reply:
I have added a parameter for choosing the namespace to install ALB.
A single ALB is required, for multiple fusions in different namespaces. We just need to set up and ingress for each fusion deployment

Reply:
Also the docs were updated